### PR TITLE
Use xcpretty for pods tests on travis.

### PR DESCRIPTION
**Pull request by thomasvl**
Travis already has it installed, so it shouldn't have to be manually installed.

### CODE REVIEW

File: Tests/pod_integration_tests.sh [hunk 1]
```diff
@@ -13,7 +13,8 @@ readonly BUILD_CFG="$2"
 # Report then run the build
 RunXcodeBuild() {
   echo xcodebuild "$@"
-  xcodebuild "$@"
+  # xcodebuild "$@"
```
Comment (sergiocampama) [inline Tests/pod_integration_tests.sh]: remove?

File: Tests/pod_integration_tests.sh [hunk 2]
```diff
@@ -13,7 +13,8 @@ readonly BUILD_CFG="$2"
 # Report then run the build
 RunXcodeBuild() {
   echo xcodebuild "$@"
```
Comment (sergiocampama) [inline Tests/pod_integration_tests.sh]: should this be changed to reflect the new command?

Comment (thomasvl) [reply Tests/pod_integration_tests.sh]: Maybe, but with the pipes, etc. it seems less likely important and easy to make a mistake.

**Comment by thomasvl** [comment]
Revised to test for xcpretty and use it in both pods and unittests.

**Pull request merged**